File: Plugins/Aspose_Barcode_Java_for_PHP/src/aspose/barcode/WorkingWith2DBarcodes/Basic2DBarcodeFeatures/CreatingQRBarcode.php
<?php
namespace Aspose\Barcode\WorkingWith2DBarcodes\Basic2DBarcodeFeatures;

use com\aspose\barcode\BarCodeBuilder as BarCodeBuilder;
use com\aspose\barcode\Symbology as Symbology;
use com\aspose\barcode\CodeLocation as CodeLocation;
class CreatingQRBarcode{

    public static function run($dataDir=null){

        # Instantiate barcode object
        $builder = new BarCodeBuilder();

        $symbology=new Symbology();
        $builder->setSymbologyType($symbology->QR);

        $builder->setCodeText("1234567890");

        # Hide code text
        $codeLocation=new CodeLocation();
        $builder->setCodeLocation($codeLocation->None);

        $builder->setRotationAngleF(90);

        # Save the image to your system and set its image format to Jpeg
        $builder->save($dataDir . "CreatingQRBarcode.jpg");

        # Display Status
        print "Created QR Barcode Successfully.".PHP_EOL;

    }


}

File: Plugins/Aspose.BarCode Java for Jython/asposebarcode/WorkingWith2DBarcodes/Basic2DBarcodeFeatures/CreatingQRBarcode.py
from asposebarcode import Settings
from com.aspose.barcode import BarCodeBuilder
from com.aspose.barcode import Symbology
from com.aspose.barcode import CodeLocation

class CreatingQRBarcode:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWith2DBarcodes/Basic2DBarcodeFeatures/CreatingQRBarcode/'
        
        # Instantiate barcode object
        builder =  BarCodeBuilder()

        symbology= Symbology
        builder.setSymbologyType(symbology.QR)

        builder.setCodeText("1234567890")

        # Hide code text
        codeLocation= CodeLocation
        builder.setCodeLocation(codeLocation.None)

        builder.setRotationAngleF(90)

        # Save the image to your system and set its image format to Jpeg
        builder.save(dataDir + "CreatingQRBarcode.jpg")

        # Display Status
        print "Created QR Barcode Successfully."


if __name__ == '__main__':        
    CreatingQRBarcode()